agents-manage-ui/src/components/sidebar-nav/app-sidebar.tsx

```diff
@@ -47,9 +46,13 @@ const bottomNavItems: NavItemProps[] = [
   },
 ];
 
-export function AppSidebar(props: React.ComponentProps<typeof Sidebar>) {
+interface AppSidebarProps extends ComponentProps<typeof Sidebar> {
+  open: boolean;
+  setOpen: Dispatch<boolean>;
+}
+
+export const AppSidebar: FC<AppSidebarProps> = ({ open, setOpen, ...props }) => {
```

Comment:
i've renamed this file from `sidebar-nav` to `app-sidebar` to keep it same as Component name